src/components/App/App.js

```diff
@@ -3,26 +3,56 @@ import React, { Component } from "react";
 import Header from "../Header/Header";
 import MovieContainer from "../MovieContainer/MovieContainer";
 import movieData from "../../mockData/movieData";
+import Hero from "../Hero/Hero";
+import { fetchMovieData } from "../../apiCalls";
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
+      allMovies: movieData.movies,
+      movies: [],
+      movieForHero: {},
+      error: "",
```

Comment:
I like how you're using empty states here (array, object, string).  Its a good way to show what shape the data will take once its there - good dev empathy.